vanilla-framework/vanilla-dashboard-theme
# List tree pattern

## Done

Added the list tree pattern as per https://github.com/vanilla-framework/vanilla-dashboard-theme/issues/9

## QA

- Pull down the branch, and do ./run build
- View the example http://0.0.0.0:8102/vanilla-dashboard-theme/examples/patterns/list-tree/
- Check it against https://github.com/ubuntudesign/vanilla-dashboard-theme-design/tree/master/List%20tree
